src/components/Code.tsx

```diff
@@ -196,7 +196,7 @@ const lightStyles = StyleSheet.create({
 });
 
 function createKey(index: string, value: string) {
```

Comment:
At this point `createKey` isn't really doing anything and you can remove the function.

Reply:
Also doesn't appear to be called anymore ;)

Reply:
Just updated! Forgot to delete.